apps/api/src/service/auth.service.ts

```
@@ -1,7 +1,7 @@
 import { Prisma, type Role } from "@prisma/client";
 import bcrypt from "bcryptjs";
 import jwt from "jsonwebtoken";
-import { ConflictError } from "../common/errors/app-error.js";
+import { ConflictError , UnauthorizedError } from "../common/errors/app-error.js";
```

ESLint

[error] Delete `·`

(prettier/prettier)